gulpfile.js

```diff
@@ -19,11 +19,8 @@ var b = browserify(opts);
 
 // add transformations here
 b.transform(reactify);
-b.transform("babelify", {presets: ["es2015", "react"]});
+b.transform("babelify", {presets: ["env", "react"]});
```

Comment:
Can this not get wrong this change what js compile as. and the .ENV file is not versioned. So with this you can chose how the fronted is compiled? Is I overthinking this?

Reply:
I have very lite experience with babel so correct me if im off. When babel started complaining about versions i arrived here: https://babeljs.io/docs/en/env/ which said:

 "By default, it has the same behavior as previous presets to compile ES2015+ to ES5"

also checking the git repo https://github.com/babel/babel-preset-env/tree/1.x

"Without any configuration options, babel-preset-env behaves exactly the same as babel-preset-latest (or babel-preset-es2015, babel-preset-es2016, and babel-preset-es2017 together)."

So I assumed that since there was no tracked .ENV or config, there would be no changes in how the frontend compiled, just various optimizations and bugfixes.

Reply:
Ah see what you mean. It's a possible security fault, but as i see it if a hostile element can modify the server or if the host is malicious then it is already game over.

Reply:
babel-preset-env has no relation to dotenv, or am I wrong about this?

Reply:
I thought so as well, okay then this is no problem at all.